packages/List/src/list-item.scss

```diff
@@ -188,6 +285,10 @@
 .ids-list-item__svg-container {
   color: var(--ids-list-item-svg-color);
   margin: var(--ids-list-item-svg-margin);
+
+  .ids-list-item--disabled & {
```

Comment:
I think that & is not useful here

Reply:
yes it is needed because the style is applied to the svg container this way.